src/components/appLayout.tsx

```diff
@@ -17,59 +17,31 @@ import Message from './message';
 import TokenValue from './tokenValue';
 
 import '../style.css';
-import Bridge from '../stores/bridge';
 import Tokens from '../stores/tokens';
 import Account from '../stores/account';
 import Web3Store from '../stores/web3';
 
-function getBridgeSuffix(bridge: Bridge, currentAddr: string) {
-  if (
-    currentAddr &&
-    bridge.defaultAddress.toLowerCase() !== currentAddr.toLowerCase()
-  ) {
-    return currentAddr;
-  }
-
-  return '';
-}
 
 interface AppLayoutProps {
-  bridgeAddr?: string;
-  bridge?: Bridge;
   tokens?: Tokens;
   account?: Account;
   web3?: Web3Store;
   section: string;
 }
 
-@inject('tokens', 'account', 'web3', 'bridge')
+@inject('tokens', 'account', 'web3')
 @observer
 class AppLayout extends Component<AppLayoutProps, any> {
   constructor(props) {
     super(props);
-    props.bridge.address = props.bridgeAddr || props.bridge.defaultAddress;
```

Comment:
Why you've removed this feature?

Reply:
There is no single bridge anymore. We would need 3 addresses for the same feature now. We might be able to replicate this feature by having a config in the UI where we can set the addresses.

Reply:
what's the conclusion here about removing the feature, guys?